src/DssSpell.t.sol

```diff
@@ -232,10 +232,10 @@ contract DssSpellTest is DSTest, DSMath {
         // Test for spell-specific parameters
```

Comment:
I think `TokenCompatibleUSDT` interface at L31 can be removed as specific for previous spell